content/blog/2022-11-07-openssl.md

```diff
@@ -0,0 +1,90 @@
++++
+tags = ["openssl", "security", "community"]
+topics = ["Security", "Release"]
+authors = ["mathieu-tortuyaux"]
+title = "About the handling of embargoed security issues"
+draft = false
+date = "2022-11-07T14:00:00+02:00"
+postImage = "/flatcar-openssl-embargo.JPG"
+aliases = ["/blog/2022/11/flatcar-openssl-embargo/"]
++++
+
+**TL; DR**: Flatcar is safe against recent OpenSSL vulnerabilities
+
+With the recent OpenSSL vulnerabilities [CVE-2022-3786][CVE-2022-3786] and [CVE-2022-3602][CVE-2022-3602],
+the Flatcar team has provided as soon as possible a batch of releases for impacted Flatcar channels (all except LTS which is not impacted).
+Releases have been published within one hour after the official public [announcement][openssl-announcment] and
+users were able to secure their workloads almost immediately.
+
+A few hours after, we started to get some user feedback regarding the OpenSSL version displayed by the CLI:
```

Comment:
```suggestion
A few hours after the release, we started to get some user feedback regarding the OpenSSL version displayed by the CLI:
```
Last nit: Not sure if it's really needed or if `few hours later, we` is better… would need to ask a native speaker if just `after, we` is also ok.

Reply:
I like `the release`, because it is explicit.